packages/dart/lib/src/hub.dart

```diff
@@ -573,6 +576,25 @@ class Hub {
     return NoOpSentrySpan();
   }
 
+  Span startSpan(
+    String name, {
+    SpanParent parentSpan = const SpanParent.active(),
```

Comment:
Since the API has the name as `parentSpan`, maybe we should also call the type `ParentSpan` to be more consistent.

Reply:
I'm thinking of calling it something like ParentSpanOption